cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -30,6 +30,249 @@ Cask Data Application Platform Release Notes
+
+- :cask-issue:`CDAP-12084` - Allow appending to (or overwriting) a PartitionedFileSet's partitions when using DynamicPartitione
+
+- :cask-issue:`CDAP-12085` - Expose a programmatic API to leverage Hive's functionality to concatenate a partition of a PartitionedFileSet.
+
+- :cask-issue:`CDAP-12094` - Enhanced add field transform plugin to add multiple fields
+
+- :cask-issue:`CDAP-12100` - Removes automatic grant/revoke privileges on CDAP entity creation/deletion.
+
+- :cask-issue:`CDAP-12134` - Applies the new authorization policy for CDAP.
+
+- :cask-issue:`CDAP-12317` - Added a new configuration 'security.authorization.extension.jar.path' in cdap-site.xml which can be used to add extra classpath and is avalible to cdap security extensions
+
+- :cask-issue:`CDAP-12367` - Support authorization on Kerberos principal for impersonation
+
+- :cask-issue:`CDAP-12378` - Workflow now allows adding configurable conditions with the lifecycle methods.
+
+- :cask-issue:`CDAP-4874` - CSD now performs a version compatibility check with the active CDAP Parcel
+
+- :cask-issue:`CDAP-12017` - Added capability to salt the row key of the metrics tables so that writes are evenly distributed and there is no region hot spotting 
+
+
+Bug Fixes
+---------
+
+- :cask-issue:`CDAP-12103` - Removed cdap-etl-realtime artifact
+
+- :cask-issue:`CDAP-12123` - Removed deprecated cdap-etl-batch jar from packaging
+
+- :cask-issue:`CDAP-12290` - Added global "CDAP" config to enable/disable metrics emission from user programs.By default metrics is enabled.
+
+- :cask-issue:`CDAP-9150` - Allow user to override the InputFormat class and OutputFormat class of a FileSet at runtime.
+
+- :cask-issue:`CDAP-12285` - Fixes an issue with the order of HBase compatibility libraries in the class path.
+
+- :cask-issue:`CDAP-9125` - CDAP Sentry Integration does not rely on every user having their own individual group
+
+- :cask-issue:`CDAP-11095` - Added support for a description field in a pipeline config that will be used as the application's description if set.
+
+- :cask-issue:`CDAP-12020` - Reuse network connections for TMS client.
+
+- :cask-issue:`CDAP-12135` - Deprecated createFilter() and added a new isVisible API in AuthorzationEn
+
+- :cask-issue:`CDAP-12143` - Removes the existing hierarchal authorization model from CDAP
+
+- :cask-issue:`CDAP-12226` - Added an optional delimiter property to the HDFS sink to allow users to configure the delimiter used to separate record fields.
+
+- :cask-issue:`CDAP-12283` - Removed version for artifacts for authorization policy to be consistent with applications. From 4.3 onwards CDAP does not support policies on artifact/application version. 
+- :cask-issue:`CDAP-12298` - Individual system service status API no longer has to goes through CDAP master.
+
+- :cask-issue:`CDAP-8629` - Allow programs to have concurrent runs in integration test cases.
+
+- :cask-issue:`CDAP-9224` - Adds upgrade step when importing a pipeline configuration
```

Comment:
doesn't seem like a bug? Also, seems like this is already covered above. Remove?